packages/terra-clinical-result/src/resulttimeheadercell/ResultTimeHeaderCell.module.scss

```diff
@@ -0,0 +1,53 @@
+@import '../common/mixins';
+
+:local {
+  .clinical-result-time-header-cell {
+    align-items: baseline;
+    display: flex;
+    flex-direction: column;
+    max-width: 100%;  // Needed for IE10 truncation
+    overflow: hidden; // Needed for IE10
+
+    .date {
+      @include terra-clinical-result-cell-truncate;
+      color: var(--terra-clinical-result-time-header-cell-date-color, rgba(28, 31, 33, 0.65));
+      flex: 0 1 auto;
+      font-size: var(--terra-clinical-result-time-header-cell-date-font-size, 0.92857em); // Must use ems for font rescaling
+      font-weight: var(--terra-clinical-result-time-header-cell-date-font-weight, 400);
+      line-height: var(--terra-clinical-result-time-header-cell-date-line-height, 1.38461);
+      max-width: 100%;
+      min-height: var(--terra-clinical-result-time-header-cell-date-min-height, 1.38461em); // Set to same as line-height in ems
+      min-width: 10px; // Needed for IE10
+
+      &.hide-date {
+        @include terra-clinical-result-visually-hidden-text;
+        min-height: var(--terra-clinical-result-time-header-cell-date-min-height, 1.38461em); // Set to same as line-height in ems
```

Comment:
I don't think we need to define this additional style. It's already being applied by the date class